taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
```

Comment:
very sensible suggestions. I can use the TACA utils for mailing and it would be nice to avoid to heavy dependencies. This is probably something for you guys in Sthlm to decide though..